app/src/main/java/com/e/salesapp/ProductListFragment.java

```diff
@@ -0,0 +1,48 @@
+package com.e.salesapp;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.e.salesapp.R;
+import com.e.salesapp.adapters.ProductAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductListFragment extends Fragment implements ProductAdapter.OnProductClickListener {
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View root = inflater.inflate(R.layout.fragment_product_list,container,false);
+        RecyclerView rvMovies = root.findViewById(R.id.rv_products);
+        List<String> productList = new ArrayList<>();
+
+        //fake data
+        productList.add("Fruits");
```

Comment:
I think this data is just for show something in the UI view, but in the last implementation please try to move these values to a resource file